test/utils/run_tests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+PATH=$PATH:node_modules/.bin
```

Comment:
This will be useless on non-symlink systems, e.g. MediaWiki-Vagrant where dependencies are installed without script symlinks. IMHO, a better variant would be to set the concrete path to the scripts we need, e.g.

```sh
mod_dir=$(cd ${0} && pwd)/node_modules
mocha=${mod_dir}/mocha/bin/mocha
```

*etc*, and then use that instead.